_docs-sources/iac/usage/composing-your-own-service.md

```diff
@@ -0,0 +1,193 @@
+# Composing your own Service
+
+We offer a collection of [services](../overview/services.md#what-is-a-service) that piece together individual [modules](../overview/modules.md#what-is-a-module) for specific use cases such as EKS clusters and VPCs with public and private subnets. While we strive to make our service catalog as a complete as possible, you may need to create your own service to suit a specific use case or need for your company.
+
+In this guide, you will learn how to create a service that provisions a simple API using the [AWS Lambda Function](../../reference/modules/terraform-aws-lambda/lambda/) and [API Gateway](../../reference/modules/terraform-aws-lambda/lambda-http-api-gateway/) modules from the Gruntwork Infrastructure as Code (IaC) Library.
+
+## Prerequisites
+
+- An AWS account with permissions to create the necessary resources
+- An [AWS Identity and Access Management](https://aws.amazon.com/iam/) (IAM) user or role with permissions to create AWS IAM roles, Lambda functions, and API Gateways
+- [AWS Command Line Interface](https://aws.amazon.com/cli/) (AWS CLI) installed on your local machine
+- [Terraform](https://www.terraform.io) installed on your local machine
+
+## Creating the service
```

Comment:
The verbs in the section and sub-section headings change. I'd probably stick to the imperative form since that seems to be the more common one here.
```suggestion
## Create the service
```